emlearn/eml_benchmark.h

```diff
@@ -6,7 +6,9 @@
 
 #if defined (__unix__) || (defined (__APPLE__) && defined (__MACH__))
 // Unix-like system
+#ifndef _POSIX_C_SOURCE
```

Comment:
duplicate of this patch in PR#18 (it was required with the new compiler configuration)